libkineto/src/output_json.cpp

```diff
@@ -38,6 +38,37 @@ void ChromeTraceLogger::handleTraceStart(
   "schemaVersion": {},
   )JSON", kSchemaVersion);
 
+#ifdef HAS_CUPTI
+  const std::vector<cudaOccDeviceProp>& occProps = KINETO_NAMESPACE::occDeviceProps();
```

Comment:
make sure the occDeviceProps call in kernelOccupancy could leverage the metadata as well to fix make the O(n) call to be O(1)?